api-documentation/import.md

```diff
@@ -50,6 +50,7 @@ Import successfully retrieved.
   "prepaid_amount_currency": "EUR|GBP|CHF",
   "settled_amount_cents": 456,
   "settled_amount_currency": "EUR|GBP|CHF",
+  "expected_invoices_count": 789,
```

Comment:
Here there are only 2 invoices returned, shouldn't `expected_invoices_count` return `2` ?

```suggestion
  "expected_invoices_count": 2,
```

Reply:
If the request is correctly done, yes. I'll update it

Reply:
We are actually returning 3 invoices here, not 2